root/static/scripts/edit/ExampleRelationships.js

```diff
@@ -103,7 +104,7 @@ searchUrl = function (mbid) {
 }
 
 
-RelationshipSearcher = function () {
+const RelationshipSearcher = function () {
```

Comment:
You also still need to change `ViewModel` to `const ViewModel` above :)

Reply:
Thank you, that is fixed too!